tests/unit/Http/CookieFactoryTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Tests\unit\Http;
+
+use Flarum\Foundation\Config;
+use Flarum\Http\CookieFactory;
+use Flarum\Tests\unit\TestCase;
+
+class CookieFactoryTest extends TestCase
+{
+    protected function factory(array $config = null): CookieFactory
+    {
+        $config = new Config(array_merge([
+            'url' => 'http://flarum.test'
+        ], $config ?? []));
+
+        return new CookieFactory($config);
+    }
+
+    /** @test */
+    public function can_create_cookies()
+    {
+        $cookie = $this->factory()->make('test', 'australia');
+
+        $this->assertEquals('flarum_test', $cookie->getName());
+        $this->assertEquals('australia', $cookie->getValue());
+        $this->assertEquals(0, $cookie->expiresAt());
```

Comment:
Isn't `expiresAt` an absolute time value, not a relative lifetime? And if so, wouldn't this cause the cookie to expire immediately?

Do we need to do any formatting into the [Date format](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Date), or is that handled by the cookie library?

Reply:
It is handled in the lib.